spec/index.js

```diff
@@ -73,6 +73,7 @@ app.whenReady().then(async () => {
     )
   );
   mocha.suite.beforeEach(function () {
+    // TODO(clavin): add support for disabling suites by title
```

Comment:
Doesn't the following line do exactly what is described here?

Reply:
Ah, here I mean just the test's own title, not the entire title chain; that is, disable by `.title` instead of `.fullTitle`. I see the ambiguity though 😅